htdocs/comm/propal/card.php

```diff
@@ -823,8 +823,15 @@
 				$type = $prod->type;
 				$fk_unit = $prod->fk_unit;
 			} else {
-				$pu_ht = price2num($price_ht, 'MU');
-				$pu_ttc = price2num(GETPOST('price_ttc'), 'MU');
+				if (!empty($conf->multicurrency->enabled)) {// we have to preserve all precision for the conversion as editline
```

Comment:
Hi @eldy ,

I take a current case, propal in USD, rate 1.27.

I add a free line of product, 100$ = 78,74015748 €
On add, pricenum(... 'MU') was apply, so i have at end : 78,74016€ et  100,0000032$, not the price i want to sell in USD.
I can update the line, on update there is no pricenum(... 'MU'), so i obtain a non rounding base price of 78,74015748€ and currency price 100$

Problem, i can correct amount, but when i create bill from propal, amounts change again. It's why i propose the same test into price.lib.php

Of course, another way is to set MU precision to high into params but i think lot of people will report a incorrect bug due to this.